packages/app/prisma-utils/src/isolation_level/isolationLevel.ts

```diff
@@ -0,0 +1,8 @@
+type IsolationLevel =
+  | 'Serializable'
+  | 'ReadCommitted'
+  | 'ReadUncommitted'
+  | 'RepeatableRead'
+  | 'Snapshot'
+
+export type CockroachDbIsolationLevel = Extract<IsolationLevel, 'ReadCommitted' | 'Serializable'>
```

Comment:
Prisma-generated code has its enum for this, so maybe we can avoid to expose this